Common/src/CConfig.cpp

```diff
@@ -8952,6 +8971,15 @@ su2double CConfig::GetWall_HeatFlux(string val_marker) const {
   return Heat_Flux[0];
 }
 
+su2double CConfig::Get_Emissivity(const string& val_marker) const {
```

Comment:
Here in CConfig.cpp look for `Roughness_Height` and `Kind_Wall`, it's used to specify roughness on no-slip boundaries for the SA model. Bonus points if you make the existing code a tiny bit less copy-pastie (you'll see what I mean)